### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ArdaGurcan**
Fix typos in core and language plugin readme files

### CODE REVIEW

File: source/plugins/languages/README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ For better results, it's advised to add either your surnames and eventually no-r
     plugin_languages_colors: "0:orange, javascript:#ff0000, ..."  # Make most used languages orange and JavaScript red
     plugin_languages_aliases: "JavaScript:JS, TypeScript:TS, ..." # Customize languages names with aliases
     plugin_languages_details: bytes-size, percentage              # Additionally display total bytes size and percentage
-    plugin_la nguages_threshold: 2%                               # Hides all languages less than 2%
+    plugin_languages_threshold: 2%                               # Hides all languages less than 2%
```
Comment (Nixinova) [inline source/plugins/languages/README.md:58]:
```suggestion
    plugin_languages_threshold: 2%                                # Hides all languages less than 2%
```
There should be another space to keep it aligned

**Comment by ArdaGurcan** [comment]
@Nixinova added space, it's ready to be merged

**Review by lowlighter — APPROVED** [review]

**Comment by lowlighter** [comment]
Thanks for your contribution!

**Comment by ArdaGurcan** [comment]
No problem, thanks for the great repo 😄

**Pull request merged**